frontend/src/components/ui/ActionModal/ActionModal.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import './actionModal.css';
+import { Button } from '../Button';
+import { useEffect } from 'react';
+const ActionModal = ({
+  title,
+  subTitle,
+  content,
+  cancelLabel = 'Cancel',
+  cancelAction,
+  submitLabel,
+  submitAction,
+  isLoading = false,
+}) => {
+  useEffect(() => {
```

Comment:
please take a look into `useLockBodyScroll` hook be already have and use it here